CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-# 1.6.x (????-??-??)
+# 1.4.0 (2017-03-14)
+## Improvements
+- Compatibilty with PIM 1.7
+
```

Comment:
Please take example on the 1.3.0 description to enrich this migration description